packages/editor/src/editor-ui/exercises/solution-feedback.tsx

```diff
@@ -0,0 +1,33 @@
+import { useInstanceData } from '@serlo/frontend/src/contexts/instance-context'
+import type { ReactNode } from 'react'
+
+export interface SolutionFeedbackProps {
+  correct: boolean
+  children?: ReactNode
+  missedSome?: boolean
+}
+
+export function SolutionFeedback(props: SolutionFeedbackProps) {
```

Comment:
Solution is usually what we call the `sample solution` authors provide. So to avoid ambiguity, I'd like to avoid it here. Maybe `Feedback` or `ExerciseFeedback`?

Reply:
`ExerciseFeedback` sounds great, way better than `SolutionFeedback` even without the `sample solution` context 😁